db/migrations/bindata.go

```diff
@@ -150,6 +150,8 @@
 // mysql/000074_upgrade_users_v6.3.up.sql
 // mysql/000075_alter_upload_sessions_index.down.sql
 // mysql/000075_alter_upload_sessions_index.up.sql
+// mysql/000076_upgrade_users_v6.6.down.sql
+// mysql/000076_upgrade_users_v6.6.up.sql
```

Comment:
Why 6.6? :p Version is not really necessary but if anything it would probably be 6.5 at this point.

Reply:
@streamer45 Thanks I wasn't sure what version to state. I suppose since we have a unique prefix (compared to any other potential `upgrade_users` migrations), we can remove the version number. I was following the convention of `db/migrations/mysql/000074_upgrade_users_v6.3.down.sql`